prisma/schema.prisma

```diff
@@ -130,7 +130,46 @@ model Campaign {
   campaignUpdateHistory CampaignUpdateHistory[]
   user                  User?                   @relation(fields: [userId], references: [id])
   userId                Int?                    @map("user_id")
+  topIssues             TopIssue[]
+  campaignPositions     CampaignPosition[]
 
   @@index([slug])
   @@map("campaign")
 }
+
+model Position {
+  id                String             @id @default(cuid())
```

Comment:
lets be consistent with the integer ids
id Int @id @default(autoincrement()) for all of these

Reply:
Ah ok, I saw your use of it in your PR so I thought we might change everything to use cuid. I'll change it back